plugins/br.com.embraer.massif.commandevaluation.commands/.settings/org.sonar.ide.eclipse.core.prefs

```diff
@@ -0,0 +1,7 @@
+analyseLocally=false
```

Comment:
Again, do we need sonar configuration. Especially if it is incorrect, e.g. see serverUrl.